src/components/Argument/components/ActionMenu.js

```diff
@@ -123,6 +123,25 @@ function ActionMenu(props) {
     );
   }
 
+  function getEdit(settings) {
+    return (
+      <button
+        className={'h5p-category-task-popover-actionmenu-edit'}
+        aria-label={settings.title}
```

Comment:
Is the `aria-label` necessary? Will it not be set by the `.h5p-category-task-popover-actionmenu-labeltext` span?

Reply:
Correct, we don't need it